kairon/chat/handlers/channels/clients/whatsapp/dialog360.py

```
@@ -41,6 +45,43 @@ def send_action(self, payload, timeout=None, **kwargs):
+            url = f'{self.app}/messages'
+
+            async with RetryClient(raise_for_status=False, retry_options=retry_options) as client:
+                async with client.post(url, json=payload, headers=self.auth_args ) as response:
+                    last_status_code = response.status
+                    if response.status == 200:
+                        resp = await response.json()
+                        return True, response.status, resp
+                    else:
+                        try:
+                            last_response = response.json()
+                        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)